org.eclipse.cbi.targetplatform.tests/src/main/java/org/eclipse/cbi/targetplatform/tests/TestGrammar.xtend

```diff
@@ -71,19 +71,19 @@ class TestGrammar {
 		val iu0 = fisrtLocation.ius.head
 		assertEquals("org.eclipse.emf.sdk.feature.group", iu0.ID)
 		assertEquals("[2.9.0,3.0.0)", iu0.version)
-		
-		val iu1 = fisrtLocation.getIus().last
+
+		val iu1 = fisrtLocation.getIus().lastOrNull
```

Comment:
This is `@since 2.35` but the MANIFEST is  `org.eclipse.xtext.xbase.lib;bundle-version="2.14.0"` so that not ideal.

I ran into this with Xcore too, but there I just defined a tiny utility method and used that so that I didn't need to change the lower bound.

Here we should do the same or we should change the lower bound.

Reply:
repo.eclipse.org was down yesterday, therefore I missed the cbi upgrade. I converted the last() calls to guava Iterables.getLast(..., null) now, which is semantically the same as lastOrNull, and does not require any manifest changes.